### ISSUE TITLE
Can the latest be published to NPM registry please?

### ISSUE THREAD

**Issue by ciaranarcher** [AlyxRen/prototype.node.js#5]
The latest in the registery (0.0.5) is missing some Hash fixes. Due to deployment concerns I'd prefer not to use the package direct from git (though that does work).

If someone could publish the latest to http://npmjs.org/ I'd certainly appreciate it. I'd rather not have to fork the project and do it.

**Comment by AlyxRen** [comment]
I will publish it when I get home. Thanks for pointing this out.
  On Jun 14, 2012 10:41 AM, "Ciaran Archer" <
  reply@reply.github.com>
  wrote:

**Comment by ciaranarcher** [comment]
Cheers!

**Comment by ciaranarcher** [comment]
Just a little reminder.

**Comment by AlyxRen** [comment]
Sorry I tried to publish it once, but was having trouble with the authentication. 

That, combined with issues in my personal/medical life, and my birthday being today, and it kinda got back-burner'd.

tldr; been neglecting you, sorry, will get to it tomorrow.

**Comment by ciaranarcher** [comment]
No rush at all, thanks for your help and the project. 

Oh, and Happy Birthday! :) 

On 28 Jun 2012, at 18:51, Stephen Middletonreply@reply.github.com wrote:

> Sorry I tried to publish it once, but was having trouble with the authentication. 
> 
> That, combined with issues in my personal/medical life, and my birthday being today, and it kinda got back-burner'd.
> 
> tldr; been neglecting you, sorry, will get to it tomorrow.
> 
> Sent from my iPod
> 
> On Jun 27, 2012, at 8:30, Ciaran Archerreply@reply.github.com wrote:
> 
> > Just a little reminder.
> > 
> > ---
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/Rixius/prototype.node.js/issues/5#issuecomment-6603228
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Rixius/prototype.node.js/issues/5#issuecomment-6636643
